components/desktop/mate/mate-session-manager/mate-session-manager.p5m

```diff
@@ -35,8 +41,8 @@ file path=usr/bin/mate-session-properties
 file path=usr/bin/mate-session-save
 file path=usr/bin/mate-wm
 file path=usr/libexec/mate/mate-session-check-accelerated
-#file path=usr/libexec/mate/mate-session-check-accelerated-gl-helper
-#file path=usr/libexec/mate/mate-session-check-accelerated-gles-helper
```

Comment:
While thinking more about this topic I start to suspect it should be enough to uncomment these two lines (without adding `pkg.depend.bypass-generate`) and run `gmake REQUIRED_PACKAGES`.  No manual `depend` line with `nvidia`/`mesa` should be needed.